cmd/res/configuration.toml

```diff
@@ -71,6 +71,9 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   AutoProvision = "true"
   Deliver = "new"
   DefaultPubRetryAttempts = "2"
+  [MessageQueue.Topics]
+  CommandRequestTopic = "edgex/command/request/#"         # subscribing for inbound command requests
```

Comment:
Actually, this should have the device service name, otherwise it will receive request for all device services.
```
  CommandRequestTopic = "edgex/command/request/device-virtual/#"         # subscribing for inbound command requests
```